SDL2/src/thread/n3ds/SDL_systhread.c

```diff
@@ -49,15 +49,21 @@ static void ThreadEntry(void *arg)
 
 int SDL_SYS_CreateThread(SDL_Thread *thread)
 {
-    s32 priority;
+    s32 priority = 0x30;
+    int cpu = -1;
     size_t stack_size = GetStackSize(thread->stacksize);
+
     svcGetThreadPriority(&priority, CUR_THREAD_HANDLE);
 
+    // hack to put mixer thread on system core
+    if(stack_size == 64 * 1024 && R_SUCCEEDED(APT_SetAppCpuTimeLimit(30)))
```

Comment:
Please use the `/* ... */` style comments here.

Reply:
Done